gen/http/openapi3.yaml

```diff
@@ -6,6 +6,8 @@ info:
 servers:
 - url: http://localhost:8000
   description: fuseml-core hosts the core services
+- url: http://0.0.0.0
```

Comment:
Why is this here? Shouldn't the url be flexible? I don't see why there should be two servers there.

Reply:
This code is auto-generated and this comes from the Host definitions on the on `design/api.go`

Reply:
I expected this. Still, it makes no sense to have those two :) 
It probably should be linked to an env variable or other.